src/reducers/profile-view.js

```diff
@@ -609,12 +624,12 @@ export const selectorsForThread = (
     );
     const getRangeSelectionFilteredThread = createSelector(
       getFilteredThread,
-      getProfileViewOptions,
-      (thread, viewOptions): Thread => {
-        if (!viewOptions.selection.hasSelection) {
+      getSelection,
```

Comment:
This change improves the ability to memoize this selector. By doing this the viewport does not need to be reinitialized for every selection of a call node within the flame graph.

Reply:
Nice fix!